Sunrise/de.lproj/Localizable.strings

```diff
@@ -175,4 +175,16 @@
 /* Get Directions */
 "Get Directions" = "Anfahrt";
 
+/* Discounts */
+"Discounts" = "Rabatt";
+
+/* Add discount code message */
+"Add your discount code bellow:" = "Fügen Sie Ihren Rabatt Code unten hinzu:";
+
+/* Discount */
+"Discount" = "Rabatt";
+
+/* No discounts */
+"No discounts - please redeem a code." = "Keine Rabatte - bitte erlöschen Sie einen Code.";
```

Comment:
FYI: there can be discounts applied to the cart without a code being added.   I haven't been able try due to lack of data in sunrise (or I don't know what to add to the cart), but you may want to check with @mmoelli  on the wording and logic.  
Discount Codes are just an optional extra condition for a cart discount.